### ISSUE TITLE
Admin Alerts: Audit headings

### ISSUE THREAD

**Issue by kraftbj** [Automattic/jetpack#2822]
We're using `<h4>` for admin alert headings, which isn't ideal for accessibility. 

h/t @afercia ( https://wordpress.slack.com/archives/accessibility/p1444241669000224 )

cc: @davidakennedy as my goto for a11y questions :)

**Comment by jeherve** [comment]
Related: #2591

And https://make.wordpress.org/core/2015/07/31/headings-in-admin-screens-change-in-wordpress-4-3/

> After the release of WordPress 4.3, the accessibility team will continue making changes to heading so that they are semantically correct. Current `<h3>`s will become `<h2>`s, `<h4>`s will become `<h3>`s, and so on.

I think this is still an ongoing process (you can follow tickets [tagged with `#a11y-headings` in Core](https://core.trac.wordpress.org/search?q=%23a11y-headings&noquickjump=1&ticket=on)), but it'd be nice if Jetpack was up to date when all of Core is updated, in 4.4 I assume.

**Comment by davidakennedy** [comment]
Yep, this is very ongoing work. :smile: 

@kraftbj Is there a screenshot where I can see what you mean by "admin alert headings," just so I know which  headings you mean?

**Comment by kraftbj** [comment]
@davidakennedy Sure thing. From @afercia, https://cldup.com/uy3P2utOi3.png

**Comment by davidakennedy** [comment]
Gotcha! So I guess to be good going forward, we'd want to move those `h4` to a `h2`.
